mlir/include/mlir/Conversion/Passes.td

```diff
@@ -16,7 +16,7 @@ include "mlir/Pass/PassBase.td"
 // ToEmitC
 //===----------------------------------------------------------------------===//
 
-def ConvertToEmitCPass : Pass<"convert-to-emitc"> {
```

Comment:
I'm confused because this PR says "Add pass ...." but here you're just using an existing pass?

Can you update the PR title and provide a description explaining the changes?